### PR TITLE
chore(sentry): bump sentry to v5

### PR DESCRIPTION
**Pull request by araujobarret**
### Description

Bumps sentry to v5.9.1 in order to unblock #9189 

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- 

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- bump sentry to v5: araujobarret

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by araujobarret** [comment]
@gkartalis and I tested on Android and iOS using the betas, everything looks fine 👍

File: src/app/Components/Lists/AuctionResultListItem.tsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ const AuctionResultListItem: React.FC<Props> = ({
               onError={() => {
                 addBreadcrumb({
                   message: `Failed to load auction result image for id: ${auctionResult.internalID}`,
-                  level: Severity.Info,
+                  level: "info",
```
Comment (gkartalis) [inline src/app/Components/Lists/AuctionResultListItem.tsx:101]: 😲 Did they remove the Severity?

**Review by gkartalis — APPROVED** [review]
🔥🥇

File: src/app/Components/Lists/AuctionResultListItem.tsx (hunk 1 above)
Comment (araujobarret) [reply src/app/Components/Lists/AuctionResultListItem.tsx:101]: yep, now the type it's just a template literal

Comment (gkartalis) [reply src/app/Components/Lists/AuctionResultListItem.tsx:101]: 😅 I liked the severity, anyways!

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (bump sentry to v5: araujobarret)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 88cca58f8ff4a14e46197ff2b7c69b8d38ae23f9
</p>

**Pull request merged**